PS8lite.Rmd

```diff
@@ -186,8 +187,37 @@ MiscVal + MoSold + YrSold" %>%
 
 Fit one Random Forest model `model_rf` using any approach you want.
 
-```{r}
+```{r eval=FALSE}
+fit_control <- trainControl(method = "cv", number = 10)
+
+# Setup search grid of "tuning" parameters
+mtry <- 2:15
+tunegrid <- expand.grid(.mtry = mtry)
 
+model_rf_caret <- caret::train(
+  form = model_formula,
+  data = training, 
+  method = "rf",
+  metric = "RMSE",
+  trControl = fit_control,
+  tuneGrid = tunegrid
+  )
+model_rf_caret
+```
+
+* 36 seems to be a large number of variables to predict one outcome, and there are some variables that overlap. Also becasue making R run through `mtry` between 2 to 36 takes forever, I decided to use 15 as an arbitrary number of measures to find the optimal number of predictor variables. 
```

Comment:
Indeed! This is a great example of a problem where *computational constraints* become an issue! In other words, method A might yield "better predictions" than method B, but if method A takes a week to run and method B takes only a few minutes, you might sacrifice a little predictive power for speed!